packages/utils/models.js

```diff
@@ -4,15 +4,24 @@ const ThisStore = {
   _storedModel: undefined,
 };
 
+/**
+ * @description Check if method is available via Mamba
+ * @returns {boolean}
+ */
 function VerifyMethodOnSystemWrapper(method) {
+  if (!__POS__) return null;
+
   try {
     /* Necessary because the circular dependency with simulator */
     const _system = window.$System || window.System;
 
-    return _system[method]();
-  } catch (error) {
-    if (__DEV__) console.error(error);
+    // Caso ele encontre o método, retorna imediatamente o valor dela
+    if (typeof _system[method] === 'function') return _system[method]();
+  } catch (_) {
+    // Cair aqui é esperado, então não faz nada. que vai pra linha seguinte.
   }
+
+  return null;
```

Comment:
Tive que mudar pra undefined, porque se o método do back retornar false !value, ele vai cair na lista estática, que é o que não queremos.

```suggestion

  return undefined;
```